dotnet/src/SemanticKernel.Core/TemplateEngine/Blocks/CodeBlock.cs

```diff
@@ -71,7 +70,7 @@ public override bool IsValid(out string errorMsg)
     }
 
     /// <inheritdoc/>
-    public async Task<string> RenderCodeAsync(Kernel kernel, ContextVariables variables, CancellationToken cancellationToken = default)
+    public async Task<string> RenderCodeAsync(Kernel kernel, IDictionary<string, string> arguments, CancellationToken cancellationToken = default)
```

Comment:
I will change this to take `KernelFunctionArguments`

Reply:
Yep, it will be done shortly after this PR in scope of this issue - https://github.com/microsoft/semantic-kernel/issues/3761